### PR TITLE
[no-jira] Allow BeanAssert to work with Joda DateTime

### PR DESCRIPTION
**Pull request by clicktravel-basharat**
This allow us to use Joda DateTime within our persistence model instead of forcing String/long conversion.

Mockito cannot mock a DateTime, so we need to put in a given (random) value.

### CODE REVIEW

File: commons/commons-test/src/main/java/com/clicktravel/common/test/BeanAssert.java [hunk 1]
```diff
@@ -106,6 +107,7 @@ private static Object getPropertyValue(final Class<?> propertyClass) {
         supportedPropertyTypes.put(Character.class, randomChar());
         supportedPropertyTypes.put(String.class, randomString(1000));
         supportedPropertyTypes.put(Date.class, new Date(randomDateTime().getMillis()));
+        supportedPropertyTypes.put(DateTime.class, randomDateTime());
```
Comment (clicktravel-robin) [inline commons/commons-test/src/main/java/com/clicktravel/common/test/BeanAssert.java]: is the plan to keep to string conversion for LocalDateTime etc?

Comment (clicktravel-basharat) [inline commons/commons-test/src/main/java/com/clicktravel/common/test/BeanAssert.java]: Not specifically. What other Joda date/time objects do we use?

**Comment by clicktravel-basharat** [comment]
Closing. Needs a bit more thought especially with the DynamoDb ReflectorUtils...

File: commons/commons-test/src/main/java/com/clicktravel/common/test/BeanAssert.java (hunk 1 above)
Comment (clicktravel-robin) [inline commons/commons-test/src/main/java/com/clicktravel/common/test/BeanAssert.java]: i belive travel use local objects to track things such as flights and check in times as they are not subject to timezones

**Pull request closed without merging**